content/docs/1.6.0/high-availability/recover-volume-when-read-only.md

```diff
@@ -0,0 +1,13 @@
+---
+  title: Recover Volume When It Becomes Read Only
+  weight: 1
+---
+
+A volume is converted to read-only state after IO requests encountered IO errors which can be caused by a network disconnection, high disk latency, abnormal node shutdown and so on.
+Longhorn periodically detects the status of the volume and automatically deletes the workload pod if the pod is managed by a controller (e.g. deployment, statefulset, daemonset, etc...) when the volume becomes read only.
+By deleting the pod, its controller restarts the pod and Kubernetes handles volume reattachment and remount.
+
+If you don't want Longhorn to automatically delete the workload pod, you can set it in [the setting `Automatically Delete Workload Pod when The Volume Is Detached Unexpectedly`](../../references/settings#automatically-delete-workload-pod-when-the-volume-is-detached-unexpectedly) in Longhorn UI.
+
+For the pods that don't have a controller, Longhorn doesn't delete them because if Longhorn does, no one will restart them.
+To recover read only volumes, you would have to manually delete and recreate the pods that don't have a controller.
```

Comment:
```suggestion
Longhorn does not delete pods without a controller because such pods cannot be restarted after deletion. To recover read-only volumes, you must manually delete and restart the pods that are not managed by a controller.
```